src/components/ArchivePage/TagPageTemplate.js

```diff
@@ -78,16 +78,20 @@ function TagPageTemplate({ pageContext }) {
 				<div className={classes.quarterItem} key={quarter}>
 					<Typography variant='h5'>{quarter}</Typography>
 					<div className={classes.quarterEvent}>
-						{quarterEventsTags[quarter].map(event =>
-							<EventInfoItem
+						{quarterEventsTags[quarter].map(event => {
+							const quarterAndYearAndEvent = quarter + ' ' + event.name;
+							const slug = quarterAndYearAndEvent.replaceAll(' ', '-').toLowerCase();
```

Comment:
The issue with this right now, is that it replaces the url with the archive page, but we're currently on the tag page. We want the same behavior with archive page, but with the tag page too.